avocado/core/job.py

```diff
@@ -264,6 +264,8 @@ def _set_output_plugins(self):
             for klass in self.args.test_result_classes:
                 test_result_instance = klass(self)
                 self.result_proxy.add_output_plugin(test_result_instance)
+        else:
+            self.result_proxy.add_output_plugin(result.Result(self))
```

Comment:
This is actually an important bugfix as without HTML support and with either json or xunit output to stdout it produces no results...